test/mocks/src/user.mock.js

```diff
@@ -110,6 +114,11 @@ export default class UserMock extends User {
     nock(this.client.apiHostname, { encodedQueryParams: true })
       .post(`${this.pathname}/_logout`)
       .reply(204);
+      
+    if (LiveServiceFacade.isInitialized()) {
```

Comment:
I covered the part about calling `unregistered()` if `LiveService` was initialized, upon logout. Tests also exist for the state of `LiveService` before and after initialization. Alos, there is only one `LiveService` instance.